code/game/gamemodes/malfunction/Malf_Modules.dm

```diff
@@ -592,7 +592,8 @@
 		active = FALSE
 		return
 	var/turf/T = get_turf(owner_AI.eyeobj)
-	new /obj/machinery/transformer/conveyor(T)
+	var/obj/machinery/transformer/TR = new(T)
+	TR.masterAI = owner_AI
```

Comment:
Can maybe be added to the `New` of the transformer. Bonus points if you refactor the `New` to `Initialize`

Reply:
Yeah that will be cleaner, I'll change that